db-server/index.js

```diff
@@ -7,6 +7,7 @@ var safeJsonFormatter = require('restify-safe-json-formatter')
 var bufferize = require('./lib/bufferize')
 var version = require('../package.json').version
 var errors = require('./lib/error')
+const RECOVERY_CODE_COUNT = 8
```

Comment:
`COUNT` seems like the wrong suffix here, unless I misunderstood. Should it be `LENGTH`?

Reply:
Nope, I misunderstood. It should be `COUNT`. 😊

Reply:
Will we ever want to change this from `8`? Any case to be made for it coming from config?

Reply:
> Will we ever want to change this from 8

👍🏽, the procedure is flexible so I'll push passing the count from the auth-server.